digitaloceanmodule.php

```diff
@@ -331,6 +331,30 @@ public function getPackageFields($vars = null) {
         return $fields;
     }
 
+	public function getallImagesDropdown($package) {
```

Comment:
Method names are expected to be camel-case, so "All" would be capitalized in the name.